Formula/c/cahute.rb

```diff
@@ -0,0 +1,26 @@
+class Cahute < Formula
+  desc "Library and set of utilities to interact with Casio calculators"
+  homepage "https://cahuteproject.org/"
+  url "https://ftp.cahuteproject.org/releases/cahute-0.2.tar.gz"
+  sha256 "70e7f12747bee8b5be7c29ecbb0c447b7c54708c3cb31fb141396776dbbe9eee"
+  license "CECILL-2.1"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libusb"
+  depends_on "sdl2"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+    bin.install "build/p7os"
```

Comment:
Why install this one separately?

Reply:
It was accidentally left out of the CMakeLists.txt in release 0.2, fixed in [a commit right after](https://gitlab.com/cahuteproject/cahute/-/commit/157f710ccd99ac6eb2595dec136f0b4a93a8bcbb). We will remove that line in the next release.

Reply:
Maybe just use the commit as a patch?

Reply:
I've done that now, but it causes `brew audit --new cahute` to fail.